src/server/stat_job.c

```diff
@@ -112,16 +112,15 @@ svrcached(attribute *pat, pbs_list_head *phead, attribute_def *pdef)
 		(server.sv_attr[(int)SRV_ATR_show_hidden_attribs].at_val.at_long == 0)) {
 		return;
 	}
+	if (pat->at_flags & ATR_VFLAG_MODCACHE)
+		/* free old cache value if the value has changed */
+		free_svrcache(pat);
```

Comment:
If you free pat here, won't you just dereference it on lines 120/122?

Reply:
thanks for catching that!